src/notifications/index.js

```diff
@@ -30,10 +27,13 @@ const {
  * Manages the notification sent for expiration files
  */
 class ExpirationNotification extends NotificationView {
+  #hasMultipleExpiredFiles
+
   constructor(options) {
     super(options)
     this.currentDate = options.currentDate
     this.filesInfo = options.filesInfo
+    this.#hasMultipleExpiredFiles = options.filesInfo.length > 1
```

Comment:
Jamais de problème avec filesInfo undefined ?

Reply:
Non, nous n'arrivons pas ici si `filesInfo` est `undefined`

Reply:
J'ai enlevé les chaînages optionnel sur `this.filesInfo` qui pouvaient induire en erreur